sdk/playwrighttesting/microsoft-playwright-testing/samples/v1/typescript/customising-service-parameters/playwright.service.config.ts

```diff
@@ -0,0 +1,24 @@
+import { getServiceConfig, ServiceOsConstants } from "@azure/microsoft-playwright-testing";
+import { defineConfig } from "@playwright/test";
+import config from "./playwright.config";
+import { AzureCliCredential } from "@azure/identity";
+
```

Comment:
We could ad comments in this file to explains things and how to play around with options or disable reporter, etc

Reply:
We will need to add README for these files